src/main/java/athena/Commands.java

```diff
@@ -12,9 +12,8 @@ public Commands() {
 
```

Comment:
Consider renaming the "Commands" class to "Command" as this relates to a single command.